lib/src/lib/components/form-fields/date-year-month-field/date-year-month-field.component.html

```diff
@@ -0,0 +1,42 @@
+<mat-form-field
+  [formGroup]="group"
+  *ngIf="!fieldIsHidden"
+  class="lab900-date-field"
+  id="lab900-date-field-{{ fieldAttribute }}"
+>
+  <mat-label *ngIf="schema.title" translate>{{ schema.title }}</mat-label>
+  <input
+    #input
+    matInput
+    placeholder="{{ placeholder | translate }}"
+    [matDatepicker]="picker"
+    [formControl]="formControl"
+    [formControlName]="fieldAttribute"
+    [required]="fieldIsRequired"
+    [readonly]="fieldIsReadonly"
+    [max]="maxDate"
+    [min]="minDate"
+  />
+  <mat-datepicker-toggle
+    [disabled]="fieldIsReadonly || fieldControl.disabled"
+    matSuffix
+    [for]="picker"
+  ></mat-datepicker-toggle>
+  <mat-datepicker
+    [disabled]="fieldIsReadonly || fieldControl.disabled"
+    #picker
+    [startView]="startView"
+    (monthSelected)="monthSelectedHandler($event, picker)"
```

Comment:
Isn't it possible to add this handler to the normal DatePicker and check for a setting to execute this?

Reply:
Well, yes it is, but this alone wouldn’t solve it, this is kinda of a workaround to be able to have a year/month date picker, the other key aspect of this is the the provider settings. So i could do this in my project but it would require a lot of custom code because this provider settings is only relevant for one part of the project, I discussed with Senn and we agree that its cleaner to just make this part of the lib directly.

Reply:
Ok, fair enough